packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -613,7 +613,6 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     return this.getErrorPayloadForClientErrorCode({
       clientErrorCode: UNKNOWN_ERROR,
       serviceErrorCode: UNKNOWN_ERROR,
-      payloadOverrides: rawError.payloadOverrides,
```

Comment:
why is this change needed?

Reply:
This was added recently into beta. We have cherry-picked the commit only today morning. https://github.com/webex/webex-js-sdk/pull/3525. WIll update the branch.